### PR TITLE
Fix compatibility issue with WTForms >= 3.0.0

### LINKED ISSUE #5: Not compatible with latest WTForms (3.0.0)
Got this error with WTForms-3.0.0:

  File "/usr/local/lib/python3.9/site-packages/wtforms/form.py", line 208, in __call__
    return type.__call__(cls, *args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/flask_wtf/form.py", line 73, in __init__
    super().__init__(formdata=formdata, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/wtforms/form.py", line 280, in __init__
    super().__init__(self._unbound_fields, meta=meta_obj, prefix=prefix)
  File "/usr/local/lib/python3.9/site-packages/wtforms/form.py", line 48, in __init__
    field = meta.bind_field(self, unbound_field, options)
  File "/usr/local/lib/python3.9/site-packages/wtforms/meta.py", line 28, in bind_field
    return unbound_field.bind(form=form, **options)
  File "/usr/local/lib/python3.9/site-packages/wtforms/fields/core.py", line 395, in bind
    return self.field_class(*self.args, **kw)
  File "/usr/local/lib/python3.9/site-packages/wtf_tinymce/forms/fields.py", line 49, in __init__
    super(TinyMceField, self).__init__(
TypeError: __init__() got an unexpected keyword argument '_name'

Apparently, parameter '_name' needs to be removed.

If this package is still maintained, and a new release (in pip repo) can be done, I'm willing to make a merge request.

### PR DESCRIPTION
**Pull request by mbtronics**
Fixes #5

### CODE REVIEW

File: wtf_tinymce/forms/fields.py [hunk 1]
```diff
@@ -107,7 +106,7 @@ def process_formdata(self, value_list):
                 attributes=self.sanitize_attributes
             )
 
-        if self.linkify:
+        if self.linkify and self.data:
```
Comment (azsoftware) [inline wtf_tinymce/forms/fields.py:109]: In which cases self.data can be None here?

Comment (mbtronics) [reply wtf_tinymce/forms/fields.py:109]: I got it during unit tests:

  File "/usr/local/lib/python3.9/site-packages/wtforms/form.py", line 208, in __call__
    return type.__call__(cls, *args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/flask_wtf/form.py", line 73, in __init__
    super().__init__(formdata=formdata, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/wtforms/form.py", line 286, in __init__
    self.process(formdata, obj, data=data, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/wtforms/form.py", line 127, in process
    field.process(formdata, data, extra_filters=field_extra_filters)
  File "/usr/local/lib/python3.9/site-packages/wtforms/fields/core.py", line 328, in process
    self.process_formdata(self.raw_data)
  File "/opt/project/app/webapp/wtf_tinymce/forms/fields.py", line 111, in process_formdata
    self.data = bleach.linkify(self.data)
  File "/usr/local/lib/python3.9/site-packages/bleach/__init__.py", line 131, in linkify
    return linker.linkify(text)
  File "/usr/local/lib/python3.9/site-packages/bleach/linkifier.py", line 173, in linkify
    raise TypeError("argument must be of text type")
TypeError: argument must be of text type

Comment (mbtronics) [reply wtf_tinymce/forms/fields.py:109]: In wtforms/fields/core.py, process_formdata is called with self.raw_data, which can be an empty list if there is something in the form (formdata), but not the field we need:

        if formdata is not None:
            if self.name in formdata:
                self.raw_data = formdata.getlist(self.name)
            else:
                self.raw_data = []

            try:
                self.process_formdata(self.raw_data)
            except ValueError as e:
                self.process_errors.append(e.args[0])
                
At this point, self.data is None and the error is triggered.

Comment (mbtronics) [reply wtf_tinymce/forms/fields.py:109]:
```
from flask_testing import TestCase
from flask_wtf import FlaskForm
from webapp.wtf_tinymce.forms.fields import TinyMceField
from flask import Flask


class TestWtfTinymce(TestCase):

    def create_app(self):
        app = Flask(__name__)
        app.config['TESTING'] = True
        app.config['SECRET_KEY'] = 'secret'
        return app

    def test_raises(self):
        class TestForm(FlaskForm):
            a = TinyMceField('a')
            b = TinyMceField('b')

        with self.app.test_request_context(path='/', method='POST', data={'a': 'sdfsdf', 'b': 'foo'}):
            TestForm()

        with self.app.test_request_context(path='/', method='POST', data={'a': 'sdfsdf'}):
            TestForm()
```

Comment (azsoftware) [reply wtf_tinymce/forms/fields.py:109]: Thank you very much for detailed explanation and problem solving.

**Pull request merged**